.github/workflows/reusable-docs.yml

```
@@ -1,29 +1,8 @@
-name: Docs
+name: 📝
```

Let's keep words (re: https://github.com/python/cpython/pull/97533#discussion_r1178495428):
```suggestion
name: Docs
```